security: Set expectation on when to get a response (#24346)
solana-labs/solana: SECURITY.md

email to security@solana.com and provide your github username so we can add you
to a new draft security advisory for further discussion.

> Expect a response as fast as possible, within 72 hours at the latest.

Reviewer: ```suggestion
Expect a response as fast as possible, typically within 72 hours.
```
